### DISCUSSION TITLE
local network Vosk server?

### DISCUSSION THREAD

**Discussion by stuckwi** [PhilippeRo/IBus-Speech-To-Text#4] [Q&A]
Does this project work on a Windows client with a Vosk server hosted on a local network?

**Comment by PhilippeRo** [comment]
This project only works locally by making use of the gst-vosk plugin developped specifically for it (that itself uses vosk C library and loads models). There is though a gstreamer plugin supporting vosk servers written in rust and I might add support in the future, but it's not a priority for the moment.
As for Windows, I have no idea but I would guess it does not since as far as I know, ibus does not run on windows.

**Comment by stuckwi** [comment]
Thank you for responding.
